PlatePath.API/Data/Models/Forum/Comment.cs

```diff
@@ -0,0 +1,25 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace PlatePath.API.Data.Models.Forum;
+
+public class Comment
+{
+    [Key]
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public int Id { get; set; }
+    
+    public int PostId { get; set; }
+    
+    public int UserId { get; set; }
+    
+    public int ParentCommentId { get; set; }
+
+    public List<Comment> ChildComments { get; set; } = new();
+
+    public List<Like> Likes { get; set; } = new();
+    
+    public int NumberOfLikes { get; set; }
```

Comment:
remove setter and make getter to return the number of items in Likes collection

NumberOfLikes => Likes.Count();